chapters/en/slides/module3/module3_04.md

```diff
@@ -3,141 +3,188 @@ type: slides
+
+- Bad messages:
+    - fixes an error
+    - added new code
+- Good messages:
+    - Reorder sections to make the README more welcoming
+    - Fix notebook error to allow for export with images
 
 ---
+
 ## JupyterLab: Making the commit
 
 
 <center>
 
-<img src='/module3/vc-commits-jupyter-5.png' width="90%" alt="404 image"/>
-
-<center/>
+<img src='/module3/vc-commits-jupyter-5.png' width="80%" alt="404 image"/>
 
+</center>
 
-Notes: **3. After recording a commit, the staging area should be empty.**
 
-After “committing” the file(s), you will see there are 0 “Staged” files. You are now ready to push your changes to the remote repository on GitHub
+Notes:
+After recording a commit, the staging area should be empty and you will see that there are 0 staged files. You are now ready to push your changes to the remote repository on GitHub, which we will show you how to do in the next slide deck.
 
 ---
+
 ## JupyterLab: Making the commit
 
 
 <center>
 
 <img src='/module3/vc-commits-jupyter-5_2.png' width="90%" alt="404 image"/>
 
-<center/>
-
+</center>
 
-Notes: If the file has been committed, you are going to be able to detect it in the History tab. You have tracked this change and now part of the history of your project!
+Notes:
+After the changes have been successfully committed, you can view them in the History tab together with your message. You have now tracked this change and made it part of the history of your project!
 
 ---
-## JupyterLab's terminal: Making the commit
+
+## Terminal: Making the commit
 
 
 <center>
 
-<img src="/module3/vc-commit-jl-terminal.png" alt="404 image" style="width:75%">
- 
-<center/> 
+<img src="/module3/vc-commit-jl-terminal.png" alt="404 image" width="70%">
 
-Notes:  In the case you don't want to add so many details and only write a title, you can use the shorter version of the command including the flag `-m`:
+</center>
 
-`git commit -m "Create an eda notebook"`
+Notes:
+The equivalent terminal command for making a commit
+with a title and long description,
+is to type `git commit` inside Bash and press enter.
+This will open your default text editor
+with a temporary file called `COMMIT_EDITMSG`
+where you can type in the commit message
+as shown in the next slide.
 
 ---
-## JupyterLab's terminal: Making the commit
-
-
-<center>
 
-<img src="/module3/vc-commit-jl-terminal-editor.png" alt="404 image" style="width:25%">
+## Terminal: Making the commit
 
 
+<center>
 
-Notes: To include a description in your commit using the terminal, you should write the command `git commit` and press ENTER. After doing that, a file called `COMMIT_EDITMSG` will open in your editor. There you can write the title and body of the commit message that you want to associate with your changes.
```

Comment:
I included this part to keep the analogy with JupyterLab. When you are using JupyterLab you can write a longer commit message if you want to.

Reply:
This is still included just above in the first slide on using the terminal, line 146 - 153.